app/ui-react/packages/ui/src/Data/Virtualizations/Views/ViewHeaderBreadcrumb.css

```diff
@@ -0,0 +1,3 @@
+.integration-detail__breadcrumb {
```

Comment:
Doesn’t this bleed in the integration editor? Why is it needed? Background colors should be set through a parent PageSection.

Also the Breadcrumb component will do same magic and inject the component in the root Page component, where the bg color is already set.

Reply:
Correct.. thanks for that. Removed.

Reply:
Logged https://issues.jboss.org/browse/TEIIDTOOLS-659 to cover moving code to functional components. I'll make sure this happens before other tabs are created. Thanks @riccardo-forina .